doc/source/configuration/host-images.rst

```diff
@@ -4,6 +4,44 @@
 Host Images
 ===========
 
+Pulling host images
+===================
+
+StackHPC provides pre-built overcloud host images through Ark, which can be
+consumed using the configuration provided by this repository.
+
+When configured, an image will be downloaded to the seed during the
+``kayobe seed service deploy`` step, and subsequently deployed using bifrost
+with ``kayobe overcloud provision``.
+
+To use these images, set ``stackhpc_download_overcloud_host_images`` to true
+in ``etc/kayobe/stackhpc-overcloud-host-images.yml``.
+
+Currently, images exist for the following operating systems:
+
+* CentOS 8 Stream
+* Rocky Linux 8
+* Rocky Linux 9
+* Ubuntu Focal 20.04
+* Ubuntu Jammy 22.04
+
+The image to download is selected automatically using the ``os_distribution``
+and ``os_release`` variables.
```

Comment:
Might be nice to mention that the images are versioned and where the versions are stored.

Reply:
Done